plugins/services/src/js/service-configuration/PodNetworkConfigSection.js

```diff
@@ -25,6 +25,13 @@ function getNetworkTypes(networks) {
   return networks.map(({ mode }) => NETWORK_MODE_NAME[mode]).join(", ");
 }
 
+function getNetworkName(networks) {
+  if (!networks || !networks.length) {
+    return null;
+  }
+  // need to get the networks to return here for network name
```

Comment:
@nLight what is the `key` name that needs to be mapped to return the network name